ccd-definition/CaseTypeTab/CareSupervision/9_Documents.json

```diff
@@ -40,7 +40,7 @@
     "TabDisplayOrder": 9,
     "CaseFieldID": "documentTabFurtherEvidenceHeading",
     "TabFieldDisplayOrder": 4,
-    "FieldShowCondition": "hearingFurtherEvidenceDocuments=\"*\" OR furtherEvidenceDocuments=\"*\" OR furtherEvidenceDocumentsLA=\"*\""
+    "FieldShowCondition": "hearingFurtherEvidenceDocuments!=\"\" OR furtherEvidenceDocuments!=\"\" OR furtherEvidenceDocumentsLA=\"*\""
```

Comment:
Do you need to make the same change to `furtherEvidenceDocumentsLA`?

Reply:
removed this as discussed when using the caseDetailsMap.putIfNotEmpty this removes need for change here